src/WebJobs.Extensions.DocumentDB/Trigger/CosmosDBTriggerAttributeBindingProvider.cs

```diff
@@ -144,5 +160,55 @@ private bool TryReadFromSettings(string settingsKey, string defaultValue, out st
             settingsValue = _nameResolver.Resolve(settingsKey);
             return !string.IsNullOrEmpty(settingsValue);
         }
+
+        private static async Task CreateIfNotExistAsync(IDocumentDBService service, string databaseName, string collectionName, int throughput)
```

Comment:
Can you make this `internal` and write some unit tests to make sure the flow works as expected (i.e. -- if the db does/doesn't exist and the collection does/doesn't exist). And to make sure throughput is properly applied. I think there's similar tests using Moq elsewhere that you can mostly copy from.